services/server/src/workers/background.ts

```diff
@@ -34,7 +34,7 @@ async function sendAuthTokenEmail({
 }) {
   const error = await safe(async () => {
     const payload: IToken.AuthTokenEmail = { type, user };
-    const token = jwt.sign(payload, jwtSecret, { expiresIn: "30m" });
+    const token = jwt.sign(payload, jwtSecret, { expiresIn: "24h" });
```

Comment:
This way, you will set all jwt expiry time to 24hr not just the confirmation email.

Reply:
```suggestion
    const token = jwt.sign(payload, jwtSecret, { expiresIn: type === IToken.Type.VerifyEmail ? "24h" : "30m" });
```